Sources/Identity/CustomerInfo.swift

```diff
@@ -45,6 +45,12 @@ import Foundation
      */
     @objc public let nonSubscriptions: [NonSubscriptionTransaction]
 
+    /// Whether the entitlements were validation.
+    ///
+    /// ### Related Symbols
+    /// - ``EntitlementValidation``
+    @objc public var entitlementValidation: EntitlementValidation { self.entitlements.validation }
```

Comment:
I'd go with "trusted" since it feels easier to wrap your head around (also, Trusted Entitlements as the official feature name has a nice ring to it)

Reply:
What would you name the type?

Reply:
Hmm maybe `EntitlementTrustStatus`? Might be too long...

Reply:
We agreed on `VerificationResult`.